tests/parsers/appbio_quantstudio_designandanalysis/testdata/appbio_quantstudio_designandanalysis_QS1_Standard_Curve_example01.json

```diff
@@ -51,11 +51,11 @@
                                             "automatic cycle threshold enabled setting": false,
                                             "automatic baseline determination enabled setting": true,
                                             "baseline determination start cycle setting": {
-                                                "value": 3.0,
+                                                "value": 3,
```

Comment:
Note to reviewer: all designandanalysis changes are just this float -> int change, and is expected.